owa/app/js/components/common/header.jsx

```diff
@@ -81,34 +91,36 @@ export default class Header extends React.Component {
             <Nav className="ml-auto" navbar>
               <NavItem>
                 <Link to="" activeClassName="active">
-                                    <UncontrolledNavDropdown>
-                                        <DropdownToggle nav caret> <FaUser/>
-                                         {' ' + this.state.currentUser}
-                                        </DropdownToggle>
-                                        <DropdownMenu>
-                                          <DropdownItem header><a href="#">My Account</a></DropdownItem>
-                                        </DropdownMenu>
-                                      </UncontrolledNavDropdown>
+                  <UncontrolledNavDropdown>
+                      <DropdownToggle nav caret> <FaUser/>
+                       {' ' + this.state.currentUser}
+                      </DropdownToggle>
+                      <DropdownMenu>
+                        <DropdownItem header><a href="#">My Account</a></DropdownItem>
+                      </DropdownMenu>
+                    </UncontrolledNavDropdown>
                      
-                    </Link>
+                  </Link>
               </NavItem>
               <NavItem>
-                    <Link to="" activeClassName="active">
-                                    <UncontrolledNavDropdown>
-                                        <DropdownToggle nav caret> <FaMapMarker/>
-                                         {' ' + this.state.currentLocationTag}
-                                        </DropdownToggle>
-                                        <DropdownMenu>
-                                          {this.state.locationTags.map(location =><DropdownItem header><a href="#">{location.display}</a></DropdownItem>)}
-                                        </DropdownMenu>
-                                      </UncontrolledNavDropdown>
-                     
-                    </Link>
+                <Link to="" activeClassName="active">
+                  <UncontrolledNavDropdown>
+                      <DropdownToggle nav caret> <FaMapMarker/>
+                       {(this.state.currentLocationTag != "")
+                                ? this.state.currentLocationTag
```

Comment:
I love the way you have formatted your code and maintained consistency, also apply that here